Sources/DeckUI/DSL/ContentItems/Code.swift

```diff
@@ -5,32 +5,34 @@
 //  Created by Josh Holtz on 8/30/22.
 //
 
+import Splash
 import SwiftUI
 
 public struct Code: ContentItem {
     public let id = UUID()
     let text: String
     let enableHighlight: Bool
+    let language: ProgrammingLanguage
     
-    public init(_ text: String, enableHighlight: Bool = true) {
+    public init(_ text: String, enableHighlight: Bool = true, language: ProgrammingLanguage = .none) {
```

Comment:
```suggestion
    public init(_ language: ProgrammingLanguage = .none, _ text: String) {
```

and `enableHighlight` could be computed?

```
var enableHighlight: Bool {
  language != .none
}
```

Reply:
This was another confusing part... the `enableHighlight` allows the user to hit the up and down arrow keys to highlight individual lines of code and go through them while presenting. 

In the video above, it's the black background highlight for the lines (which also makes the font bold and removes syntax highlighting... although those are all configurable)

Reply:
ahhh gotcha

Reply:
I've updated to try something similar, but slightly different.... 

```
public init(_ language: ProgrammingLanguage = .none, enableHighlight: Bool = true, text: () -> String)
```

This allows one to do something like:

```
Code(.swift) {
"""
import Splash
import SwiftUI

public struct Code: ContentItem {
    public let id = UUID()
}
"""
}
```

Reply:
Yes, it's quite elegant 🤩